static/figure/css/figure.css

```diff
@@ -495,7 +486,7 @@
     }
     .tab-pane .ui-slider-horizontal {
         height: 10px;
-        width: 150px;
+        width: 215 px;
```

Comment:
Trivial CSS style thing but you've introduced a space before the `px` and it's the only instance of a space before `px` in this file.